gufe/settings/models.py

```diff
@@ -3,130 +3,77 @@
-    cutoff: FloatQuantity["angstrom"] = Field(None, description="Default units are in angstroms")
-    switch_width: FloatQuantity["angstrom"] = Field(None, description="Default units are in angstroms")
-    solvent_dielectric: Optional[float]
+    temperature: FloatQuantity["kelvin"] = Field(
+        None, description="Simulation temperature, default units kelvin"
+    )
+    pressure: FloatQuantity["standard_atmosphere"] = Field(
+        None, description="Simulation pressure, default units standard atmosphere (atm)"
+    )
+    ph: Union[PositiveFloat, None] = Field(None, description="Simulation pH")
+    redox_potential: Optional[float] = Field(
+        None, description="Simulation redox potential"
+    )
 
 
-class GBSASettings(SettingsBaseModel):
-    """Settings for Generalized-Born surface area (GBSA) implicit solvent parameters.
+class ProtocolSettings(SettingsBaseModel, abc.ABC):
+    """Protocol-specific settings; this is a base class for protocol
+    developers to use for building any settings not included elsewhere.
 
-    See the `SMIRNOFF GBSA specification <https://openforcefield.github.io/standards/standards/smirnoff/#gbsa>`_
-    for more details.
     """
 
-    gb_model: str = "OBC1"
-    solvent_dielectric: float = 78.5
-    solute_dielectric: float = 1
-
+    ...
 
-class ForcefieldSettings(SettingsBaseModel):
-    """Settings for the forcefield.
-    """
 
-    # Metadata
-    date: Optional[date]
-    author: Optional[str]
+class BaseForcefieldSettings(SettingsBaseModel, abc.ABC):
+    """Base class for ForcefieldSettings objects"""
 
-    # These should also allow None
-    vdW: VdWSettings
-    electrostatics: ElectrostaticSettings
-    gbsa: GBSASettings
+    ...
 
 
-class ThermoSettings(SettingsBaseModel):
-    """Settings for thermodynamic parameters.
+class OpenMMSystemGeneratorFFSettings(BaseForcefieldSettings):
+    """This class is a WIP.
 
-    No checking is done to ensure a valid thermodynamic ensemble is possible.
+    Right now we just basically just grab what we need for the systemgenerator signature
+    https://github.com/openmm/openmmforcefields#automating-force-field-management-with-systemgenerator
     """
 
-    temperature: FloatQuantity["kelvin"] = Field(None, description="Simulation temperature, default units kelvin")
-    pressure: FloatQuantity["standard_atmosphere"] = Field(None, description="Simulation pressure, default units standard atmosphere (atm)")
-    ph: Union[PositiveFloat, None] = Field(None, description="Simulation pH")
-    redox_potential: Optional[float] = Field(None, description="Simulation redox potential")
-
-
-class ProtocolSettings(SettingsBaseModel, abc.ABC):
-    """Protocol-specific settings; this is a base class for protocol
-    developers to use for building any settings not included elsewhere.
-
-    """
-    ...
+    # I don't want app.HBonds or 4*unit.amu, so will string-ify those for now
+    forcefield_kwargs: dict = {
+        "constraints": "app.HBonds",
```

Comment:
given that this is OpenMM-specific, shouldn't this have a way to either take in the OpenMM singletons or create them from a string?

Reply:
I will have to look into [`openmmforcefields`](https://github.com/openmm/openmmforcefields#automating-force-field-management-with-systemgenerator) to see what we can do, I didn't want to require `openmm` for testing which is why I left it as a string.